quantum/keymap_common.c

```diff
@@ -74,7 +74,21 @@ action_t action_for_key(uint8_t layer, keypos_t key)
     } else if (keycode == RESET) { // RESET is 0x5000, which is why this is here
         clear_keyboard();
         _delay_ms(250);
+// bootloader_jump doesn't work for the Teensy
+#ifdef TEENSY
```

Comment:
I believe the line you're referring to is commented out via the `#if 0` [here](https://github.com/jackhumbert/qmk_firmware/blob/master/tmk_core/common/avr/bootloader.c#L117).